javascript/javascript1/week1/houseypricey.js

```diff
@@ -0,0 +1,17 @@
+    
+const house1 = 8 * 10 * 10 * 2.5 * 1000 + 100 * 300;
```

Comment:
Can we put the width, height, depth values and garden size values in their own variables like:

`const height = 500;` ect.

Makes the code more readable :)